README.md

```diff
@@ -7,9 +7,85 @@ Table of Contents
 =================
 
 * [Name](#name)
+* [Synopsis](#synopsis)
 * [Description](#description)
 * [Copyright & License](#copyright--license)
 
+Synopsis
+========
+
+Basic usage
+
+```
+
+# set variable in ob file scope.
+foo = bar;  we create a variable foo which value is bar. variable in ob is string 
+replace , op will render variable like some template engine.
+
+# comment in ob file.
+#foo = bar; ob use # as comment syntax.
+
+# block in ob file.
+foo {
+    command1; 
+    command2; 
+}
+
+Block is basic execute unit.we use op compile out pl with make keyword to execute wanted unit.
```

Comment:
Basically,

```
foo {
    directive1;
    directive2;
}
```

is a *rule*. And `foo` is the target. It's quite similar to Makefile's rules per se.